docs/guides/upgrade.rst

```diff
@@ -41,17 +41,17 @@ Upgrade your node
 
 .. code-block:: bash
 
-    docker run --rm -ti \
-        -v $(pwd)/keys:/opt/pyaleph/keys \
-        -v $(pwd)/node-secret.key:/opt/pyaleph/node-secret.key:ro \
-        -v $(pwd)/config.yml:/opt/pyaleph/config.yml:ro \
-        alephim/pyaleph-node:beta \
-        python3 /opt/pyaleph/migrations/config_updater.py \
+    docker-compose [-f <docker-compose-file>] down
+
+    docker-compose [-f <docker-compose-file>] \
+            run pyaleph /opt/pyaleph/deployment/migrations/config_updater.py \
             --key-dir /opt/pyaleph/keys \
             --key-file /opt/pyaleph/node-secret.key \
             --config /opt/pyaleph/config.yml \
             upgrade
 
+    docker-compose [-f <docker-compose-file>] down
```

Comment:
down -> up -d

Reply:
The docker-compose `run` command will start the other services.
Do do the migration independently from restarting the service, I here suggest to first tear everything down before restarting it.

Reply:
Okay, if you prefer.